scripts/tokenomics-emulation/network.ts

```diff
@@ -81,6 +81,21 @@ class Network {
     return value;
   }
 
+
+  /**
+   * Fetch native token balance
+   * @param address account addess
+   * @returns raw balance
+   */
+  public async getRawBalance(address: string) {
+    console.log(`About to get balance for: ${address}`);
+    const {
+      data: { free: balance },
+    } = await this.api.query.system.account(address);
+    const formatedBalance = (+balance / 10 ** this.decimals).toFixed(10);
```

Comment:
What is the last `10`? Move it to the constant with understandable name

Reply:
Moved to config.